.github/workflows/ci.yml

```diff
@@ -54,10 +54,10 @@ jobs:
     name: Documentation
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - uses: julia-actions/setup-julia@latest
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
         with:
-          version: '1.4'
+          version: '1'
```

Comment:
It may be better to specify a particular version here, for example `1.6`.

The reason is that `jldoctest` blocks are very sensitive to formatting and the output of `Base.show` is not considered part of the stable interface for Base.

So doctests can break between Julia versions.